docs/book/products.rst

```diff
@@ -4,69 +4,113 @@
 Products
 ========
 
-**Product** model represents unique products in your Sylius store. Every product can have different variations or attributes and has following values:
+**Product** model represents unique products in your Sylius store. Every product can have different variations or attributes.
 
-* *name* - The full name of the product
-* *slug* - Urlized version of the name
-* *description* - Description of the product
-* *shortDescription* - Simple description of the product for lists and banners
-* *metaDescription* - Description for search engines
-* *metaKeywords* - SEO keywords
-* *createdAt* - Date of creation
-* *updateAt* - Timestamp of the most recent update
+.. note::
+
+   Each product has to have at least one variant to be sold in the shop.
 
 Options
 -------
 
-In many cases, you will have product with different variations. The simplest example would be a T-Shirt available in different sizes and colors.
+In many cases, you will want to have product with different variations.
```

Comment:
`will` => `would`, `product` => `a product` if i'm not mistaken that you mean singular in this context